modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3228,6 +3228,14 @@
 		"skipParametersValidation": true,
 		"to": "CPTaxCategoryCreateDateComparator.getInstance(param#0#)"
 	},
+	{
+		"classNames": [
+			"JournalArticle"
+		],
+		"from": "regex:(?<variableName>\\w+)\\.(?<prefixMethodName>setDDMStructure)Key\\((?<whitespace>(\\s+|))(?<argumentVariableName>\\w+)\\.(?<argumentPrefixMethodName>getDDMStructure)Key\\(\\)\\)",
+		"issueKey": "LPD-30559",
+		"to": "${variableName}.${prefixMethodName}Id(${whitespace}${argumentVariableName}.${argumentPrefixMethodName}Id())"
+	},
```

Comment:
@rubenssoares-br this one its wrong.

Isn't necessary to be a regex. Even if would its wrong.
What happens if the key was filled directly? Would not be a match...
Unnecessary catch for withspaces, the replacement can be inline, no worries about that.

Can be a from normal, with the key for the parameter, and the to should be a way of getting the id by the key. If possible.

Also, please check if its correct, maybe the person when this one was created wasn't 100% right about the pattern it self. Search in the liferay portal and validate like you did for the last one.